vertx-web/src/main/java/io/vertx/ext/web/handler/sockjs/impl/EventBusBridgeImpl.java

```diff
@@ -165,19 +165,19 @@ private void internalHandleSendOrPub(SockJSSocket sock, boolean send, JsonObject
       () -> {
         String address = msg.getString("address");
         if (address == null) {
-          replyError(sock, "missing_address");
+          replyError(sock, "MISSING_ADDRESS", "Message address is missing");
```

Comment:
we should define those in an package protected enum instead of havingthem scattered in the code base